backend/ebios_rm/helpers.py

```
@@ -2,6 +2,11 @@
 import math
 import random
 from global_settings.models import GlobalSettings
+from .models import EbiosRMStudy, FearedEvent, RoTo
```

Ruff (0.8.2)

`.models.EbiosRMStudy` imported but unused

Remove unused import

(F401)